### PR TITLE
Pull request: fix issue #388

### PR DESCRIPTION
**Pull request by merlindoescode**
I worked through the issue #388

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @merlindoescode I just have some comments below

File: README.md [hunk 1]
```diff
@@ -1042,17 +1054,11 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
...
-- :bulb: [Cheatsheet Collection](https://devhints.io)
+- :bulb: [Cheatsheet Collection - DevHints](https://devhints.io)
 - :bulb: [Cheat Sheets](http://www.cheat-sheets.org/)
 - :bulb: [Programming Cheat Sheets](https://cheatography.com/)
-- :bulb: [Python Cheatsheet](https://www.pythoncheatsheet.org/)
-- :bulb: [Devhints - TL;DR for developer documentation](https://devhints.io/)
-- :bulb: [Linux Commands Cheat Sheet](https://www.linuxtrainingacademy.com/linux-commands-cheat-sheet/)
 - :bulb: [Codehouse - Cheatsheet Collection](https://codehouse.vercel.app/app)
-- :bulb: [CSS Cheat Sheet](https://www.hostinger.com/tutorials/css-cheat-sheet)
+
```
[the remark this answers is not in the document]
Comment (merlindoescode) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -1099,6 +1105,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 ### Tools
 
+- :wrench: [Emmet](https://docs.emmet.io/cheat-sheet/)
```
[the remark this answers is not in the document]
Comment (merlindoescode) [reply README.md]: done

**Comment by merlindoescode** [comment]
@shahednasser 
I did all requested changes.
Thx for your time!

**Comment by shahednasser** [comment]
Awesome thanks!

**Pull request merged**